From 82dc98319f421a4685657fcff21502304448b1ae Mon Sep 17 00:00:00 2001
From: Allen Xu <b45815@freescale.com>
Date: Mon, 1 Dec 2014 19:55:18 +0800
Subject: [PATCH] MLK-9939 ARM: dts: Add ddrsmp parameter to device tree for
 i.MX6SX ARM2 board

The ddr sample point is board related, so add ddrsmp parameter to device
tree for i.MX6SX 17x17 ARM2 board.

DDRSMP value:
2 ---- i.MX6SX 17x17 ARM2 board

Signed-off-by: Allen Xu <b45815@freescale.com>
(cherry picked from commit c5d9eb443cda0c4d6e5705a2b51904f49b4f8297)
---
 arch/arm/boot/dts/imx6sx-17x17-arm2.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/imx6sx-17x17-arm2.dts b/arch/arm/boot/dts/imx6sx-17x17-arm2.dts
index 971416c..b989ba7 100644
--- a/arch/arm/boot/dts/imx6sx-17x17-arm2.dts
+++ b/arch/arm/boot/dts/imx6sx-17x17-arm2.dts
@@ -439,6 +439,7 @@
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_qspi2_1>;
 	status = "okay";
+	ddrsmp=<2>;
 
 	flash0: n25q256a@0 {
 		#address-cells = <1>;
-- 
1.8.0

